sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/DDLParserSuite.scala

```diff
@@ -723,6 +723,21 @@ class DDLParserSuite extends AnalysisTest {
     assert(exc.getMessage.contains("INSERT INTO ... IF NOT EXISTS"))
   }
 
+  test("show tables") {
+    comparePlans(
+      parsePlan("SHOW TABLES"),
+      ShowTablesStatement(None, None))
+    comparePlans(
+      parsePlan("SHOW TABLES FROM testcat.ns1.ns2.tbl"),
+      ShowTablesStatement(Some(Seq("testcat", "ns1", "ns2", "tbl")), None))
+    comparePlans(
+      parsePlan("SHOW TABLES IN testcat.ns1.ns2.tbl"),
+      ShowTablesStatement(Some(Seq("testcat", "ns1", "ns2", "tbl")), None))
+    comparePlans(
+      parsePlan("SHOW TABLES IN tbl LIKE '*dog*'"),
```

Comment:
Nit: is `*` or `%` the wildcard?

Reply:
I think it's `*`: https://github.com/apache/spark/blob/607fb879060f8e620d74017ff3fd8c8c737cdbab/sql/core/src/test/resources/sql-tests/inputs/show-tables.sql#L16